doc/development/development_on_osx.rst

```diff
@@ -154,25 +143,35 @@ If you encounter any error during the installation of Pillow, make sure that lib
     brew install libjpeg zlib
     brew link --force zlib
 
-Tribler should now be able to startup without warnings by executing this command in the Tribler root directory:
+Build
+=====
```

Comment:
We have a separate guide for this. This file should only explain how to setup the environment on MacOS.